cli/miner/actor_test.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 func TestWorkerKeyChange(t *testing.T) {
-	//stm: @OTHER_WORKER_KEY_CHANGE_001
+	// stm: @OTHER_WORKER_KEY_CHANGE_001
```

Comment:
Undo change, probably a code editor auto-format. I think these `stm:` commands needed to be carefully formatted. One day we'll probably just remove them all but for now let's leave them alone.

Reply:
Done